### ISSUE TITLE
Details for ablation study

### ISSUE THREAD

**Issue by hobin-jeong** [Junshk/FRN#2]
Hello, @Junshk ! 
I've been studying the FRN structure you proposed, and I have questions for ablation studies.

1) When you studied to get the effectiveness of down-upsampling modules, you suggested RCAN structure when down-upsampling modules were not equipped. In this case, the body part is only changed, right?

2) The next question is for auto-encoder loss. I wonder the training process if the auto-encoder loss is given. When training, I think the following cases:

- down-upsampling modules w/o FRN body are trained -> the whole model w/ FRN body is trained using the pre-trained weights for down-upsampling modules.
- down-upsampling modules w/o FRN body are trained -> the whole model w/ FRN body is trained using the trained weights for down-upsampling modules, and simultaneously the weights for down-upsampling modules are fine-tuned.
- From scratch, the whole model is trained using L1 + auto-encoder loss.

I'm strongly guessing the first case works.

Thanks in advance! :)

**Comment by Junshk** [comment]
Hi, @hobin-jeong 
1. To get the effectiveness of down-upsampling module, compare the first and the second experiments in table 2. of our paper. The structure used in the first is original RCAN with scale 1. Another structure is RCAN with down-upsampling module. For fair comparison, the body parts of those structures are same (while head and tail parts are different). 
2. We choose the third case. The auto-encoder loss is jointly optimized with L1 loss.

**Comment by hobin-jeong** [comment]
Thank you! I'll check your paper based on your reply :)
